main.go

```diff
@@ -48,8 +50,8 @@ var (
 
 func init() {
 	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
-
 	utilruntime.Must(appv1alpha1.AddToScheme(scheme))
+	utilruntime.Must(machinev1beta1.Install(scheme))
```

Comment:
NIT: Is there any reason to use `Install` rather than `AddToScheme`?

Reply:
IIUC, internally it's the same thing, and `AddToScheme` is deprecated
https://pkg.go.dev/github.com/openshift/api/machine/v1beta1#pkg-variables

Reply:
aha, then you can use `Install` for the other two appearances of `AddToScheme`

Reply:
`Install` does not exist for the other packages :)